Cake.Recipe/Content/localbuild.cake

```diff
@@ -47,33 +53,45 @@ public class LocalBuildRepositoryInfo : IRepositoryInfo
 {
     public LocalBuildRepositoryInfo(ICakeContext context)
     {
-        context.Information("Getting current branch name...");
-        IEnumerable<string> redirectedStandardOutput;
-        IEnumerable<string> redirectedError;
-
-        var exitCode = context.StartProcess(
-            "git",
-            new ProcessSettings {
-                Arguments = "branch --show-current",
-                RedirectStandardOutput = true,
-                RedirectStandardError = true
-            },
-            out redirectedStandardOutput,
-            out redirectedError
-        );
+        var gitTool = context.Tools.Resolve("git");
 
-        if (exitCode == 0)
+        if (gitTool == null)
```

Comment:
This check should probably also check if there is a .git directory as well.

Reply:
That’s a fair point, as they might have git installed, but still have chosen to download the source code